### ISSUE TITLE
[BUG]: Selenium tests not working on my page. Error: Cannot read properties of undefined 

### ISSUE THREAD

**Issue by kbpitula** [IBMa/equal-access#1356]
### Project

a11y checker

### Browser

Chrome

### Operating system

Windows

### Description

When i execute on my new page view i got that kind of error

Error message:
```
TypeError: Cannot read properties of undefined (reading 'length')
    at ACReportManager.setLevels (/Users/krzys/repos/com.ibm.itest.cloud.apsportal.featuregroup/node_modules/accessibility-checker/lib/ACReportManager.js:520:74)
    at /Users/krzys/repos/com.ibm.itest.cloud.apsportal.featuregroup/node_modules/accessibility-checker/lib/ACHelper.js:318:64
    at step (/Users/krzys/repos/com.ibm.itest.cloud.apsportal.featuregroup/node_modules/accessibility-checker/lib/ACHelper.js:33:23)
    at Object.next (/Users/krzys/repos/com.ibm.itest.cloud.apsportal.featuregroup/node_modules/accessibility-checker/lib/ACHelper.js:14:53)
    at fulfilled (/Users/krzys/repos/com.ibm.itest.cloud.apsportal.featuregroup/node_modules/accessibility-checker/lib/ACHelper.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
/Users/krzys/repos/com.ibm.itest.cloud.apsportal.featuregroup/node_modules/accessibility-checker/lib/ACReportManager.js:520
        for (var _i = 0, pageResults_1 = pageResults; _i < pageResults_1.length; _i++) {
```
```
"devDependencies": {
        "accessibility-checker": "3.1.26",
        "chromedriver": "110.0.0",
        "selenium-webdriver": "4.8.0"
    }
```

### Steps to reproduce

Link to slack: 
https://ibm-analytics.slack.com/archives/C036P1CTN/p1678701918841659

**Comment by aliunwala** [comment]
Triage:
Here is another link to this thread
https://ibm-studios.slack.com/archives/C036P1CTN/p1678701918841659

**Comment by shunguoy** [comment]
@kbpitula tested with two settings: worked in the cloud, but failed in the cluster installation. The pages used in the two environments are identical except for the element CSS classes.  The page runs fine in the cluster environment if some of those classes were removed.  
 It seems that the cause of the error is some js script failed to load. More error logs were added to the code, hopefully, the logs can provide more clues on the real cause.
 The customer is suggested to use the new build for the test to see if more errors can be caught for further debugging.  The issue will be closed for now.

**Comment by ErickRenteria** [comment]
Merged
